### DISCUSSION TITLE
Feature Request: Pinyin Search for Lawnchair Launcher

### DISCUSSION THREAD

**Discussion by kinglau66** [LawnchairLauncher/lawnchair#4116] [Ideas]
Hello Lawnchair Team,

I'm a user of Lawnchair Launcher and would like to suggest adding Pinyin search functionality. This feature would allow users, especially those who speak Chinese, to find their apps more easily by typing the Pinyin (Romanized) version of the app names.

Why It's Useful:

    Makes app searching faster for Chinese-speaking users.
    Improves accessibility and inclusivity.

Considerations:

    Impact on performance.
    Accuracy and efficiency of Pinyin conversion.
    User option to enable/disable this feature.

Implementing Pinyin search could significantly enhance usability for a large user base. I believe this addition would make Lawnchair even more appealing and user-friendly. Thank you for considering my suggestion!

(Note: This message was generated using ChatGPT by OpenAI.)

**Comment by E8Ib** [comment]
For example, "微信" (wei'xin), just type "wx" to search it.
